week-1-practise/level-3/1-piping.js

```diff
@@ -11,6 +11,7 @@
 */
 
 var startingValue = 5;
+var result = format(add(multiply(startingValue, 2), 10));
```

Comment:
Nice! Can you explain which function is evaluated first, second and third?

Reply:
Not quite right - the `multiply` function will be first, because the `add` function needs to know what numbers to add together. Then the `add` will be evaluated, and then `format` :)

Reply:
Thanks,